index.html

```diff
@@ -41,10 +41,10 @@ <h1 class="logo">My log</h1>
           <section >
               <div class="header-intro">
                   <div>
-                    <h1 class="greeting">
+                    <h2 class="greeting">
                       I'm Michael.
                       Glad to see you!
-                    </h1>
+                    </h2>
```

Comment:
- [ ] The page title should be a heading using `hx` tag which you have used. However, since this is our first title, for accessibility reasons, kindly try as much as possible to use the `h1` tag. In addition, people with poor sight use this heading to clearly understand what the page is all about ☺️.